src/backend/context.rs

```diff
@@ -488,6 +490,35 @@ impl Context {
 		Ok(())
 	}
 
+	/// Zoom a window.
+	fn zoom_window(&mut self, window_id: WindowId, zoom_delta: f32) -> Result<(), InvalidWindowId> {
+		let window = self
+			.windows
+			.iter_mut()
+			.find(|w| w.id() == window_id)
+			.ok_or(InvalidWindowId { window_id })?;
+
+		window.zoom += zoom_delta;
```

Comment:
```suggestion
		window.zoom += zoom_delta;
```

Does `+=` make sense here? It seems more logical to multiply by for example 1.1 or 0.9 rather than adding / subtracting.

But maybe it's better to just overwrite the zoom level with the given value.